app/assets/javascripts/covid19/src/webapp/components/structures-table/SearchBar.tsx

```diff
@@ -37,20 +38,16 @@ export const SearchBar: React.FC<SearchBarProps> = React.memo(props => {
     } = props;
     const [open, setOpen] = React.useState(false);
     const [stateValue, setValueDebounced] = useDebouncedSetter(value, setValue, { delay: 500 });
-    const [autoSuggestionOptions, setAutoSuggestionOptions] = React.useState<Array<string>>([
-        "6YOR",
-        "Homo sapiens",
-        "SARS-CoV-2",
-    ]);
+    const [autoSuggestionOptions, setAutoSuggestionOptions] = React.useState<Array<string>>(
```

Comment:
[veeery minor, no need to change] 1) `string[]` is more idiomatic. But more complex `Array<{x: number, y: string}>` are ok. 2) When the value is inferred, we don't usually force the type in useState.